bubble_sort.rb

```diff
@@ -0,0 +1,65 @@
+def bubble_sort(unsorted_array)
+  n = unsorted_array.length
+  i = 1
+  dummy_flag = true
+  while dummy_flag
```

Comment:
:bulb: [OPTIONAL] It is not dummy, actually it's doing a pretty great job by optimizing your code. So we can give it a proper name, maybe like `swapped?` This will help other developers to understand your code faster.